packages/terra-form-select/src/search/Menu.jsx

```diff
@@ -1,416 +1,390 @@
+      return;
     }
 
-    if (option.props.onMouseEnter) {
-      option.props.onMouseEnter(event);
+    /* no options to display */
+    if (noResults) {
+      ariaLive.innerText = intl.formatMessage({ id: 'Terra.form.select.noResults' }, { text: searchValue });
+      return;
     }
-  }
 
-  /**
-   * Scrolls the active option into view.
-   * Options already in view will not scroll.
-   * Options above the dropdown will be scrolled to the top of the menu.
-   * Options below the dropdown will be scrolled to the bottom of the menu.
-   */
-  scrollIntoView() {
-    const activeOption = this.menu.querySelector('[data-select-active]');
-    if (!this.state.active || activeOption === null) {
-      return;
+    // Detects if option is clear option and provides accessible text
+    if (clearOptionDisplay) {
+      const activeOptionElement = listbox.querySelector('[data-select-active]');
+      if (activeOptionElement && activeOptionElement.hasAttribute('data-terra-select-clear-option')) {
+        ariaLive.innerText = intl.formatMessage({ id: 'Terra.form.select.clearSelect' });
+        return;
+      }
     }
 
-    const dropdownRect = this.menu.parentNode.getBoundingClientRect();
-    const optionRect = activeOption.getBoundingClientRect();
+    if (activeOption) {
+      const isClearSelectOption = activeOption.props.display === '' && activeOption.props.value === '';
+      const activeSelected = MenuUtil.isEqual(value, activeOption.props.value);
 
-    if (optionRect.top < dropdownRect.top) {
-      activeOption.scrollIntoView();
-    } else if (optionRect.bottom > dropdownRect.bottom) {
-      activeOption.scrollIntoView(false);
+      if (isClearSelectOption) {
+        ariaLive.innerText = intl.formatMessage({ id: 'Terra.form.select.clearSelect' });
+      } else if (activeSelected) {
+        ariaLive.innerText = intl.formatMessage({ id: 'Terra.form.select.selectedText' }, { text: activeOption.props.display });
+      } else {
+        ariaLive.innerText = activeOption.props.display;
+      }
+    } else {
+      ariaLive.innerText = '';
     }
-  }
-
-  render() {
-    return (
-      /**
-       * Note: role="listbox" and aria-activedescendant needed for VoiceOver on iOS to properly
-       * shift the virtual indicator to this DOM element when dropdown is rendered. If you modify these
-       * attributes, you'll need to manually verify that the virtual indicator on iOS is still shifted
-       * to the dropdown / the selected item in the dropdown if an item is selected when the dropdown
-       * is opened.
-       */
+  });
+
+  const menuClasses = cx([
+    'menu',
+    { 'is-above': isAbove },
+    { 'is-focused': true },
+    { 'is-invalid': isInvalid },
+  ]);
+
+  /* eslint-disable react/forbid-dom-props */
+  return (
+    <div ref={menuRef} id={menuId} className={menuClasses} aria-describedby={ariaDescribedBy}>
+      <div className={cx('menu-container')}>
+        <div className={cx('display')} aria-label={ariaLabel}>
```

Comment:
I believe we'll need to add `role="application"` here. JAWS doesn't seem to honor up/down arrow key events in its default "browse" mode. Switching this over to "application" mode will allow JAWS to pass through the up/down arrow key events like we expect.